testbeam_analysis/examples/eutelescope.py

```diff
@@ -293,6 +293,7 @@ def run_analysis():
         output_tracks_file=os.path.join(output_folder, 'Tracks_all_Kalman.h5'),
         exclude_dut_hit=True,
         pixel_size=pixel_size,
+        n_pixels=n_pixels,
```

Comment:
It is required to calculate the whole sensor area which is the error on the initital state in case DUT 0 is not in fit selection.